prometheus-to-sd/main.go

```diff
@@ -60,6 +60,12 @@ var (
 		"Namespace name of the pod in which monitored component is running.")
 	zoneOverride = flag.String("zone-override", "",
 		"Name of the zone to override the default one (in which component is running).")
+	containerNameLabel = flag.String("container-name-label", "",
```

Comment:
Considering that you moved this to per source configuration, we don't need those flags anymore. Also don't forget to revert known-flags.txt file.

Reply:
done